src/bilder/images/concourse/templates/vector.yaml

```diff
@@ -1,11 +1,5 @@
 ---
 sources:
-  vector_metrics:
-    type: internal_metrics
-    scrape_interval_secs: 30
-  host_metrics:
```

Comment:
We should keep the host metrics, that's useful to have for CPU, disk, memory, etc. Particularly for worker nodes so we can understand what their performance looks like.